error.vue

```diff
@@ -1,10 +1,17 @@
 <template>
-  <!-- TODO: test and refactor -->
   <section class="error">
     <div class="m-auto">
       <Vue3Lottie :animation-data="notFoundPage" :height="200" :width="200" />
 
-      <p class="Error-text">Not found page, please go back to the home page.</p>
+      <template v-if="error.statusCode === 404">
+        <h1>404!</h1>
+        <p class="Error-text">
+          Not found page, please go back to the home page.
```

Comment:
If this component represents only 404 error I think better to rename the file to "notFoundPageError".
If this component is a global error, in this case better pass error code and description in props

Reply:
This is an error page for global errors. When the application encounters some error, users are redirected to error.vue. It is implemented by Nuxt under the hood. So it is not a typical component, it is a page with auto redirects when errors are occurring 

I work with error dynamically via this composable: 
`const error = useError();`

depends on error, users can see different templates
<img width="575" alt="image" src="https://github.com/LeonidShv/un-environment/assets/57797423/64c47c34-d275-41be-bb29-25f48235ce10">